Makefile

```diff
@@ -65,6 +65,7 @@ gotidy:
 
 .PHONY: gogenerate
 gogenerate:
+	@command -v goimports >/dev/null 2>&1 || $(MAKE) -B install-tools
```

Comment:
I dont have much context on the setup of this repo, I ll leave this to @lahsivjar or @carsonip who might be better to check this part.

Reply:
I've briefly looked at it. IMO it is an upstream mdatagen bug causing missing import line `"github.com/elastic/opentelemetry-collector-components/processor/ratelimitprocessor/internal/metadata"` in `processor/ratelimitprocessor/internal/metadatatest/generated_telemetrytest_test.go`. You may submit an issue upstream mdatagen. To avoid blocking your work, add a comment next to your goimports workaround and reference the upstream bug report. That'll be good enough

Reply:
Done, thanks for looking into it!